packages/varlet-ui/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "module": "esnext",
+    "outDir": "./dist",
```

Comment:
maybe `outDir` is unnecessary?

Reply:
Unfortunately. when we set `allowJs` as `true`. The typescript will warn the `overwrite` in console. Because `typescript` or others bundle tools who can read `tscofig.json`. Will overwrite the bundle file. Refer[Reason](https://github.com/microsoft/TypeScript/issues/14538)

Reply:
According the following issue. I think set the field `outDir` is quickest way. Or may use `exclude` option repalce the outDir?